extensions/amp-carousel/0.1/slidescroll.js

```
@@ -289,4 +373,20 @@ export class AmpSlideScroll extends BaseCarousel {
       }
     }
   }
+
+  /**
+   * Animate scrollLeft of the container.
+   * @param {number} fromScrollLeft.
+   * @param {number} toScrollLeft.
+   * @returns {!Promise}
+   */
+  animateScrollLeft_(fromScrollLeft, toScrollLeft) {
+    if (fromScrollLeft == toScrollLeft) {
+      return Promise.resolve();
+    }
+    const interpolate = numeric(fromScrollLeft, toScrollLeft);
+    return Animation.animate(this.slidesContainer_, pos => {
+      this.slidesContainer_./*REVIEW*/scrollLeft = interpolate(pos);
+    }, 80, 'ease-in-out').thenAlways();
```

won't calling `thenAlways` suppress errors?

cc @dvoytenko